src/uwtools/config.py

```diff
@@ -104,23 +106,47 @@ def dump_file(self, output_path):
         with open(output_path, 'w', encoding="utf-8") as file_name:
             yaml.dump(self.data, file_name, sort_keys=False)
 
+    class Loader(yaml.SafeLoader): #pylint: disable=too-many-ancestors
+        '''Yaml loader to add INCLUDE tag for yaml file concatenation'''
+
+        def __init__(self, stream: IO) -> None:
+            try:
+                self._root = os.path.split(stream.name)[0]
+            except AttributeError:
+                self._root = os.path.curdir
+
+            super().__init__(stream)
+
+    def add_include(loader: Loader, node: yaml.Node) -> Any: #pylint: disable=no-self-argument
+        '''method to implement INCLUDE tag for yaml file concatenation'''
+        #pylint: disable=no-member
```

Comment:
What is the linter complaining about here? There are only rare occasions where this disable is needed as this check helps us to make sure we aren't doing something we shouldn't be. 

If it is truly needed, we should document the need for it in a comment so that we don't have to guess at what's going on.